package.json

```diff
@@ -56,8 +57,9 @@
     "tus-js-client": "^2.2.0"
   },
   "dependencies": {
+    "@aws-sdk/client-s3": "^3.18.0",
     "@google-cloud/storage": "^5.3.0",
-    "aws-sdk": "^2.761.0",
+    "aws-sdk": "^2.929.0",
```

Comment:
Not sure which AWS SDK should be used. `aws-sdk` is used in the code but I think this is deprecated in favor of `@aws-sdk/client-s3˙. Need to re-check and remove one.

Reply:
I decided to remove `@aws-sdk/client-s3` as this is a complete rewrite and has diffrent API. This should be changed in separate PR.